### ISSUE TITLE
Consistant segmentation faults for vlc.Media.set_meta()

### ISSUE THREAD

**Issue by lethargilistic** [oaubert/python-vlc#14]
I've encountered segmentation faults while trying to set AlbumArtist, DiscNumber, and DiscTotal.

``` python3
python-vlc (master) » python3                                ~/Dev/python-vlc  
Python 3.5.1+ (default, Mar 30 2016, 22:46:26) 
[GCC 5.3.1 20160330] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import vlc
>>> m = vlc.Media("file://./assets/ChopinOp9No2.mp3")
>>> m.set_meta(vlc.Meta.AlbumArtist, 'AlbumArtist')
[1]    15685 segmentation fault  python3

```

``` python3
>>> import vlc
>>> m = vlc.Media("file://./assets/ChopinOp9No2.mp3")
>>> m.set_meta(vlc.Meta.DiscNumber, 'DiscNumber')
[1]    15739 segmentation fault  python3

```

``` python3
>>> import vlc
>>> m = vlc.Media("file://./assets/ChopinOp9No2.mp3")
>>> m.set_meta(vlc.Meta.DiscTotal, 'DiscTotal')
[1]    15555 segmentation fault  python3
```

**Comment by oaubert** [comment]
It works for me (linux, vlc 2.2.3, with python2 and python3). What are your OS and vlc version?

**Comment by lethargilistic** [comment]
Linux (Ubuntu 16.04) and VLC revision 2.2.2-0-g6259d80. I'll see about updating to VLC 2.2.3 and get back to you.

**Comment by lethargilistic** [comment]
I'm the worst and I forgot. I updated to VLC 2.2.5, rebased to your current master branch, and still get a seg fault.

**Comment by oaubert** [comment]
I confirm the behaviour, but it is a VLC version issue: AlbumArtist, DiscNumber, and DiscTotal are not defined in vlc 2.2. They are in the development version (and the generated/vlc.py is generated against the development version). Please use the 2.2 version of the bindings (available in the repo in generated/2.2) so that you are not mistaken by non-existing symbols in the python code.

**Comment by lethargilistic** [comment]
OK, thanks for the clarification. ^^
